common/templates/ellipses.html

```diff
@@ -1,10 +1,12 @@
-<div ng-switch="isHtml">
-    <div ng-class="{'hideContent': overflow == true, 'showContent': overflow == false}" >
-	    <span ng-switch-when="true" class="markdown-container" ng-bind-html="content"></span>
-	    <span ng-switch-default>{{content}}</span>
+<td ng-repeat="val in rowValues track by $index">
+    <div ng-switch="val.isHTML">
+        <div ng-class="{'hideContent': hideContent == true, 'showContent': hideContent == false}" >
+            <span ng-switch-when="true" class="markdown-container" ng-bind-html="val.value"></span>
+            <span ng-switch-default>{{val.value}}</span>
```

Comment:
@shinyichen You may want to use `ng-bind` here also to better avoid the problem described in #787, so that it's `<span ng-switch-default ng-bind="val.value"></span>`.

This [SO thread](http://stackoverflow.com/questions/16125872/angularjs-why-ng-bind-is-better-than-in-angular) has more info on `ng-bind` vs `{{ }}`, if you'd like to read more.